From b0e40f297fa2009bc27017f957e22da3a3fbf7cd Mon Sep 17 00:00:00 2001
From: Joe Konno <joe.konno@intel.com>
Date: Wed, 1 May 2019 12:55:04 -0700
Subject: [PATCH 2/2] schemas: dash-to-dock: do not animate show-apps

The animation for showing all applications can be costly and/or slow,
depending on the underlying platform. For Clear Linux, disable
animate-show-apps by default. End-users can change this default through
the gnome-tweaks applet or through gsettings/dconf.

Caveat: this does not impact the Show Applications animation when it's
clicked in the Activities view.

Signed-off-by: Joe Konno <joe.konno@intel.com>
---
 schemas/org.gnome.shell.extensions.dash-to-dock.gschema.override | 1 +
 1 file changed, 1 insertion(+)

diff --git a/schemas/org.gnome.shell.extensions.dash-to-dock.gschema.override b/schemas/org.gnome.shell.extensions.dash-to-dock.gschema.override
index a82e55e..b94a165 100644
--- a/schemas/org.gnome.shell.extensions.dash-to-dock.gschema.override
+++ b/schemas/org.gnome.shell.extensions.dash-to-dock.gschema.override
@@ -1,4 +1,5 @@
 [org.gnome.shell.extensions.dash-to-dock]
+animate-show-apps=false
 apply-custom-theme=false
 click-action='minimize-or-overview'
 custom-background-color=false
-- 
2.21.0

